### ISSUE TITLE
Release not created/published for Inspec Plugin

### ISSUE THREAD

**Issue by robh-wirehive** [hashicorp/packer-plugin-inspec#9]
#### Overview of the Issue

There is not a current release 0.01 or higher so this plugin for inspec does not work with Packer 1.7.3 using the plugins file with 
`packer init`

The changelog from Packer 1.7.3 indicates that this version should be available for Inspec (https://github.com/hashicorp/packer/blob/master/CHANGELOG.md)

#### Reproduction Steps

`Packer init .`

using packer.plugins.pkr.hcl
```
packer {
    required_plugins {
        inspec = {
            version = ">=v0.0.1"
            source = "github.com/hashicorp/inspec"
        }
    }
}
```

### Plugin and Packer version

1.7.3

### Operating system and Environment details

Docker image hashicorp/packer:1.7.3

**Comment by nywilken** [comment]
> Hi contributors @nywilken @sylviamoss - would you be able to create a release version tagged 0.0.1 or higher please. Thanks

Hi @robh-wirehive thanks for reaching out. This is one of the plugin we haven't had a chance to split out yet, but will try to get to it soon. 

In the meantime, the inspec plugin is still bundled with Packer so you should be able to use the plugin without needing to run `packer init`. 

If you remove it from the `required_plugins` block Packer should load the bundled plugin and work as expected. Please let me know if that is not the case for you.

**Comment by robh-wirehive** [comment]
Thanks for the update @nywilken 👍

**Comment by nywilken** [comment]
Plugin has been released as version v0.0.7.
